pyuvdata/uvcal/calfits.py

```diff
@@ -796,39 +715,26 @@ def read_calfits(
                 data = fname[0].data
                 if self.cal_type == "gain":
                     self.gain_array = (
-                        data[:, :, :, :, :, 0] + 1j * data[:, :, :, :, :, 1]
+                        data[:, 0, :, :, :, 0] + 1j * data[:, 0, :, :, :, 1]
                     )
-                    self.flag_array = data[:, :, :, :, :, 2].astype("bool")
-                    n_arrays = hdr.pop("NAXIS1")
-                    if n_arrays == 5:
-                        self.input_flag_array = data[:, :, :, :, :, 3].astype("bool")
-                        self.quality_array = data[:, :, :, :, :, 4]
```

Comment:
I should probably update the calfits memo to reflect that the `input_flag_array` is deprecated.

Reply:
I made an update to the tex file, feel free to swat it away if it's not what you are looking for, but if you think it's alright I'll recompile the PDF and commit that as well.